tokio-postgres/tests/test/main.rs

```diff
@@ -347,7 +347,250 @@ async fn custom_range() {
     assert_eq!("floatrange", ty.name());
     assert_eq!(&Kind::Range(Type::FLOAT8), ty.kind());
 }
+/// This test check to make sure that empty responses for select queries include the header but not
+/// for other query types.
+#[tokio::test]
+async fn simple_query_select_transaction() {
+    let client = connect("user=postgres").await;
+
+    // The following statements configure the tables for the test.
+    let _ = client.simple_query("DROP TABLE sbtest1").await;
+    let _ = client.simple_query("DROP TABLE sbtest2").await;
+    let _ = client
+        .simple_query("CREATE TABLE sbtest1 (id INTEGER, k INTEGER);")
+        .await;
+    let _ = client
+        .simple_query("CREATE TABLE sbtest2 (id INTEGER, k INTEGER);")
+        .await;
+    /// This test check to make sure that empty responses for select queries include the header but not
+    /// for other query types.
+    #[tokio::test]
+    async fn simple_query_select_transaction() {
+        let client = connect("user=postgres").await;
+
+        match client.simple_query("DROP TABLE sbtest1").await {
+            Ok(_) => {
```

Comment:
Can we just run `DROP TABLE IF EXISTS sbtest1` and unwrap() the result? Since the test is simple enough, I don't think printing success/failure is necessary. Same with the next command.